### DISCUSSION TITLE
How to save clean/pre-processed data?

### DISCUSSION THREAD

**Discussion by Shubham-Kr-Shaw** [pycaret/pycaret#1287] [Q&A] [answered]
As we run the following lines of code:-

from pycaret.datasets import get_data
juice = get_data('juice')
from pycaret.classification import *
exp_name = setup(data = juice,  target = 'Purchase')

Pycaret does all the data cleaning, pre-processing, feature engineering, and feature selection stuff with the default setting.
That's cool.
But how can I save and get this pre-processed data in my system?

**Comment by Yard1** [comment, marked as answer]
You can obtain it with `get_config("X")` (or "y", "X_train", "y_train", "X_test", "y_test")

**Reply by Shubham-Kr-Shaw** [reply]
Thank You
